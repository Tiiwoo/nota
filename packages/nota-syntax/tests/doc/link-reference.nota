[link][ref] [Other]

[ref]: http://example.com
[other]: http://other.com