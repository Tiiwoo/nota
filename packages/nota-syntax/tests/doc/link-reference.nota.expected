[
  el(
    "p",
    {},
    el(
      "a",
      {
        href: "http://example.com",
      },
      "link"
    ),
    " ",
    el(
      "a",
      {
        href: "http://other.com",
      },
      "Other"
    )
  ),
  null,
  null,
];